### DISCUSSION TITLE
VERY OLD Versions

### DISCUSSION THREAD

**Discussion by shawty** [cyanfish/naps2#482] [General]
So for reasons unknown to me, some folks I know have a very OLD Canon Microfilm Scanner (An MS400 to be exact).

This beast of a thing only has drivers up to XP for it, so they have a battered old non internet connected P4 based windows XP machine that they use to drive it.  It's a non profit local history group, so telling them to go out and buy new gear is not going to happen, this thing (Like everything they have) was donated to them.

Anyway.... I've managed to make thier old XP machine work again, and shock horror I even managed to find some SCSI drivers for the scanner interface, but they lack any even half decent software to actually make scans from the device.

So my question here is quite simply:

**At which version did NAPS 2 stop supporting windows XP?**

I use NAPS2 myself personally on my own newer machines, and I know it's a really great piece of software for scanners (Was my saviour when my Epson stopped printing, and at the same time refused to let me use the scanner on it too), and I know it's been around for a considerable amount of time, so I figured that at some point it may have supported XP, and finding and installing the old version for these folks would solve their problem.

So which version do I need to go back to in order to this?

Cheers
Shawty

**Comment by cyanfish** [comment]
See https://www.naps2.com/windows-scanning#system-requirements. You can try 6.1.2 and if it doesnt work, 5.8.2.

**Comment by shawty** [comment]
Hi @cyanfish I'll check those versions out, thanks for your reply.

Shawty
